pkg/server/api/auth/auth.go

```diff
@@ -29,67 +29,67 @@ type GoogleAuthenticator struct {
 	TrustedUserEmailDomain string
 }
 
-func (g GoogleAuthenticator) AuthenticateRequest(r *http.Request) (string, error) {
-	var accessToken string
-	_, err := fmt.Sscanf(r.Header.Get("Authorization"), "Bearer %s", &accessToken)
+func (g GoogleAuthenticator) AuthenticateRequest(r *http.Request) (string, string, error) {
+	var refreshToken string
+	_, err := fmt.Sscanf(r.Header.Get("Authorization"), "Bearer %s", &refreshToken)
 	if err != nil {
-		return "", fmt.Errorf("Error extracting token from Authorization header: %s", err.Error())
+		return "", "", fmt.Errorf("Error extracting token from Authorization header: %s", err.Error())
 	}
 
 	// abr uses a shared secret to authenticate
```

Comment:
It's weird we have this comment in something open-source. It's also stale?

Reply:
✂️